tikv-control.md

```diff
@@ -229,6 +229,7 @@ Use the `compact` command to manually compact data of each TiKV. If you specify
 
 - Use the `--host` option to specify the TiKV that needs to perform compaction.
 - Use the `-d` option to specify the RocksDB that performs compaction. The optional values are `kv` and `raft`.
```

Comment:
remove this

Reply:
Done in https://github.com/pingcap/docs/pull/9549/commits/a670d190157297acd121dff245bfe56350a95939